src/Kunstmaan/ApiBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,32 @@
+services:
+    kunstmaan_api.controller.nodes:
+        class: Kunstmaan\ApiBundle\Controller\NodesController
+        arguments:
+            - "@doctrine.orm.default_entity_manager"
```

Comment:
Single quotes. For the rest of the file too :)

Reply:
done